### PR TITLE
Completed Task 796 : Design System > Elements > Enhancements in Review

### LINKED ISSUE #796: Design System > Elements > Enhancements in Review
Design System > Elements > Enhancements in Review
Docs	1] Review development different than design
Default	2] Development is different than design
https://www.figma.com/design/vziFLZAgMFi8wA5SlikLh5/Raaghu-Design-System?node-id=1586-7562&node-type=frame&t=ncBHc3XyPjDjE2We-0

### PR DESCRIPTION
**Pull request by prakashmhaske**
-  **Review**
## Screenshots :
 
image

 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: raaghu-components/src/rds-comp-feeds/rds-comp-feeds.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ export interface Item {
     feedIcon?: string;
     imageUrl?: string;
     description: string;
+    descriptionShort?: string;
```
Comment (hemalatagaikwad) [inline raaghu-components/src/rds-comp-feeds/rds-comp-feeds.tsx]: @prakashmhaske  this property is not used in interface, why this is added?

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
Change Title of PR

File: raaghu-components/src/rds-comp-feeds/rds-comp-feeds.tsx (hunk 1 above)
Comment (poojagurav21) [reply raaghu-components/src/rds-comp-feeds/rds-comp-feeds.tsx]: Removed this property as this is not in use.

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Pull request merged**